src/Lykke.Job.BlockchainCashinDetector/Modules/CqrsModule.cs

```diff
@@ -13,52 +12,79 @@
 using Lykke.Job.BlockchainCashinDetector.Workflow.Sagas;
 using Lykke.Job.BlockchainOperationsExecutor.Contract;
 using Lykke.Messaging;
-using Lykke.Messaging.Contract;
 using Lykke.Messaging.RabbitMq;
 using Lykke.Messaging.Serialization;
-
+using System.Collections.Generic;
+using MoreLinq;
+using System.Linq;
 
 namespace Lykke.Job.BlockchainCashinDetector.Modules
 {
     public class CqrsModule : Module
     {
-        private static readonly string Self = BlockchainCashinDetectorBoundedContext.Name;
+        public static readonly string Self = BlockchainCashinDetectorBoundedContext.Name;
```

Comment:
Looks like there are no reason to make this field public.
Change my mind :-)

Reply:
that field is used in tests.

Reply:
![image](https://user-images.githubusercontent.com/14955144/47523184-7c850d80-d8a8-11e8-9de9-5516ac0dd1de.png)